### DISCUSSION TITLE
Should we use project boards?

### DISCUSSION THREAD

**Discussion by wehrad** [cryo-data/discuss#10] [General]
In combination with Issues? Any Github organization can open [project boards](https://github.com/orgs/cryo-data/projects) that offer more possibilities  for project management than Issues alone. Maybe could we just give it a try and see?

**Comment by mankoff** [comment]
I have not found a use for GitHub project board feature, so I'm not familiar  with them. I think that they are just an alternate display of issues, so they cannot be used `instead of`, only `in combination with`. Have you used project boards before? What is your use-case for this project?

**Comment by wehrad** [comment]
I never used it before but always found it quite interesting. E.g. existing issues from different repositories can be stored in categories, e.g. [here](https://github.com/orgs/GlacioHack/projects/2) for the GlacioHack organization planning the next Hackathon. That might become useful if we realize that for instance, we have issues related to metadata production/handling in different repositories.

**Comment by wehrad** [comment]
But in our case I think we'll not spread across many different repositories in the long run, in such a way that we need to centralize issues. So project boards might no be relevant for this project.
